sources/packages/web/src/types/contracts/institution/EducationProgramOffering.ts

```diff
@@ -46,3 +48,24 @@ export interface CourseDetails {
   courseStartDate: string;
   courseEndDate: string;
 }
+
+/**
+ * Offering bul insert validation result with formatted values
```

Comment:
typo `bul`

Reply:
![image](https://user-images.githubusercontent.com/61259237/190233317-198177c5-3a8c-4b1c-950a-16e39605bf3e.png)

Reply:
"Bul" changed to "bulk".